enterprise/backend/test/metabase_enterprise/api/collection_test.clj

```diff
@@ -0,0 +1,26 @@
+(ns metabase-enterprise.api.collection-test
```

Comment:
EE code isn't strictly organized, but generally I think we try to group things by feature flag, so this should go under the `audit_app` directory. Maybe ` metabase-enterprise.audit-app.api.collection-test`